osgar/replay.py

```diff
@@ -59,7 +59,7 @@ def replay(args, application=None):
 
     driver_name = module_config['driver']
     module_class = get_class_by_name(driver_name)
-    module_instance = module_class(module_config['init'], bus=bus)
+    module_instance = module_class(module_config.get('init', {}), bus=bus)
```

Comment:
This change probably has some good motivation and deserves to be in a separate simpler PR.

Reply:
OK, good point #611